setup.py

```diff
@@ -3,9 +3,9 @@
 setup(
     name='osipi_code_collection',
     version='0.0.2',
-    install_requires=['dicom', 'imageio', 'joblib', 'lmfit',
-                      'mat73', 'matplotlib', 'numpy', 'opencv-python', 'openpyxl',
-                      'pandas', 'progressbar', 'pytest', 'scipy'],
+    install_requires=['dicom==0.9.9.post1', 'imageio==2.19.3', 'joblib==1.1.0', 'lmfit==1.0.3',
```

Comment:
Do we want them to be exactly these versions? You can specify a range such as `pkg>=2.0,<3.2`.

Reply:
Hi @sunyu0410 I had feedback at ISMRM, suggesting it would be helpful if each version/release of our repo specified which package versions we used for the testing.
That would mean people could reproduce our test results locally.
That's why I've specified the package versions.
However, I'm open to alternative solutions...